extras_install.sh

```diff
@@ -29,6 +29,21 @@ if [ ${ID} = "raspbian" ]; then
    pip3 install opencv-contrib-python==4.1.0.25
 fi
 
+if [ ${ID} = "ubuntu" ]; then
+   pip3 install --upgrade pip
+   pip3 install opencv-contrib-python
+fi
+
+if [ ${ID} = "mendel" ]; then
+
+   git clone https://github.com/pjalusic/opencv4.1.1-for-google-coral.git /tmp/opencv_coral
```

Comment:
Please don't fetch packages from random people's github account. If there is no alternative, maybe host this internal to Dianomic.